### DISCUSSION TITLE
new discussion/idea?

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockO/uBO-Scriptlets#26] [General] [answered]
@uBlock-user 
did any of your scriptlets fix such `https://github.com/AdguardTeam/AdguardFilters/issues/161760` ie if i understand correctly ag fix replacing certain script tag with another
note= is fixed in ubo

**Comment by uBlock-user** [comment]
> if i understand correctly ag fix replacing certain script tag with another

How to do that via javascript ?

**Reply by ghajini** [reply]
can you see adguard is doing

**Reply by ghajini** [reply]
AG rule= it selects first script tag on the webpage using `document.querySelector("script")` then if a script tag is found, the function adds the class name `"atScript_0"` to the script tag using `t.classList.add("atScript_0")`

antiadb code = `pastebin.com/MQ6RXanu` ,part responsible `pastebin.com/jU7phN7F`

**Reply by uBlock-user** [reply]
This is what they're doing -- `||reshare.pm^$replace=/<script type="text\/javascript">/<script class="atScript_0" type="text\/javascript">/,document`

Translate this in English

**Reply by ghajini** [reply]
translate?

**Reply by uBlock-user** [reply]
Yes explain what is that filter doing.

**Reply by ghajini** [reply]
its replacing script with another script class tag atScript_0
if i understand

**Reply by uBlock-user** [reply]
Hmmm, so what you're asking is for a script node inserter right ?

**Reply by ghajini** [reply]
yes ,if that fixes above issue

**Reply by uBlock-user** [reply]
Why not just use a scriptlet that adds an attribute named "class" with the value "atScript_0" ?

**Reply by ghajini** [reply]
which scriptlet? working filter?

**Reply by uBlock-user** [reply, marked as answer]
https://github.com/gorhill/uBlock/wiki/Resources-Library#set-attrjs-

Unfortunately, custom value cannot be set.
